### PR TITLE
docs(CONTRIBUTING): fix typo `graps` => `grabs`

### PR DESCRIPTION
**Pull request by arturmiglio**
Just a small fix of a typo, to open the "door" for more contributions :)

In `CONTRIBUTING.md`:

> Feel like doing a bit more? You can check what is up for ~graps~**grabs** in our public backlog and create a PR for it

### CODE REVIEW

File: docs/release-notes.mdx [hunk 1]
```diff
@@ -8,6 +8,12 @@ import { TitleCard } from './components/TitleCard';
     Keep up to date with the latest releases of the design system.
 </TitleCard>
 
+## Unreleased
```
Comment (nlopin) [inline docs/release-notes.mdx:11]: We don't need to put this into release because it is technical documentation, not public

**Review by nlopin — DISMISSED** [review]

**Comment by arturmiglio** [comment]
Added another fix in the `docs/contributing` documentation page.

One small issue is that the installation instructions are not currently valid, since the package is not yet published. Or maybe it is published under a different scope with public access?

Maybe it's a good idea to note that on the docs...

**Comment by duranmla** [comment]
Thanks for contributing @arturmiglio indeed we need to make a manual release while https://github.com/freenowtech/wave/issues/38 gets solved

**Review by duranmla — APPROVED** [review]

**Comment by arturmiglio** [comment]
> Thanks for contributing @arturmiglio indeed we need to make a manual release while #38 gets solved

@duranmla does it make sense to note on the README (in case people is trying to install the package with the supplied command)?

**Comment by duranmla** [comment]
@arturmiglio I don't think so, I will try to make a release manually today if it is possible.
_The current usage of the library doesn't direct me to an actual state of urgency, but it is true we need to be coherent. Apologizes_

**Pull request merged**